lib/ActivityStreams.js

```diff
@@ -129,6 +132,18 @@ ActivityStreams.prototype = {
   },
 
   /**
+   * Sets up the toolbar button that links to the Timeline page.
+   */
+  _setupButton() {
+    ActionButton({
+      id: "activity-streams-link",
+      label: "Activity Streams",
```

Comment:
Are we going with the singular or plural of "Activity Stream"?
#askingforafriend